internals/cli/cli.go

```diff
@@ -155,7 +156,7 @@ type defaultOptions struct {
 // Parser creates and populates a fresh parser.
 // Since commands have local state a fresh parser is required to isolate tests
 // from each other.
-func Parser(cli *client.Client) *flags.Parser {
+func Parser(cli *client.Client, options *RunOptions) *flags.Parser {
```

Comment:
Another good point. I don't like just reusing `CmdOptions` because it has the parser in it. I think the better thing to do would be to add a new struct `ParserOptions` (even though it's a tiny bit more wiring), which takes the client, pebble dir, and socket path. So what about this?

```go
type ParserOptions struct {
    Client *client.Client
    PebbleDir string
    SocketPath string
}

func Parser(opts *ParserOptions) *flags.Parser {
    ...
}
```